src/test/java/com/systelab/seed/user/UserResourceTest.java

```diff
@@ -30,6 +30,7 @@ public void testGetUserList() {
         TestUtil.checkObjectIsNotNull("Users", users);
     }
 
+    // TODO: Pending login verification
     @Description("Create a User with name, login and password")
     @Test
     public void testCreateUser() {
```

Comment:
I would create additional scenario to test Create User without username, password and name.

Reply:
done